### PR TITLE
added first version for basque and catalan

### PR DESCRIPTION
**Pull request by iolalla**
Hey,

As you requested here is the simplest case for Catalan and Basque.

The demo part doesn't work cause the script looks in the path "https://raw.githubusercontent.com/snowballstem/snowball-data/master/$id/voc.txt" and this hasn't been updated yet.

Cheers.

### CODE REVIEW

File: demo.tt [hunk 1]
```diff
@@ -23,7 +23,7 @@ stemming algorithm:
 [% END %]
 <script>
 var old_words;
-function stem_demo(elt) {
+function (elt) {
```
Comment (ojwb) [inline demo.tt]: Won't this break the demo?

File: algorithms/basque/stemmer.tt [hunk 2]
```diff
@@ -1,52 +1,54 @@
-[% header('A Basque stemming algorithm') %]
+[% header('Euskera stemming algorithm') %]
```
Comment (ojwb) [inline algorithms/basque/stemmer.tt]: Is there a reason for this change?  We call this stemmer "Basque" everywhere else, and that does seem to be the English name for the language - we don't say "Deutsch stemming algorithm" for the German stemmer.

File: demo.tt (hunk 1 above)
Comment (iolalla) [reply demo.tt]: Mmmmh I wasn't councious I changed that, but I tested it with the proposed function name and still says error...

File: algorithms/basque/stemmer.tt (hunk 2 above)
Comment (iolalla) [reply algorithms/basque/stemmer.tt]: Agree my bad.

**Pull request merged**